types.ts

```diff
@@ -0,0 +1,86 @@
+export type PlayerID = number;
+
+type Player = {
+    id: PlayerID,
+    name: string,
+}
+
+type PlanetID = number;
+
+type Planet = {
+    id: PlanetID;
+    x: number;
+    y: number;
+    size: number; // just for visualizer
```

Comment:
```suggestion
    // just for visualizer
    size: number; 
```
![image](https://user-images.githubusercontent.com/5972402/202918206-58ba02b0-6218-4592-9a8d-76b14ba05aac.png)
De szóljatok ha a VSCode ezt jól értelmezi.

Reply:
A vscode nem írja oda sehova ezt.